cmd/kubeadm/app/phases/bootstraptoken/clusterinfo/clusterinfo_test.go

```diff
@@ -102,7 +118,12 @@ func TestCreateBootstrapConfigMapIfNotExists(t *testing.T) {
 					})
 				}
 
-				err := CreateBootstrapConfigMapIfNotExists(client, file.Name())
+				if tc.fileExist == true {
+					err = CreateBootstrapConfigMapIfNotExists(client, file.Name())
+
+				} else {
+					err = CreateBootstrapConfigMapIfNotExists(client, "notexistfile")
```

Comment:
call the function only one time. use an if/then branch to fill a new name variable if the file exists or not. pass this variable to the function.